Makefile

```diff
@@ -2,29 +2,36 @@
 ManyWellCMD=python examples/many_well.py -m training.seed=0,1,2 # by default seeds are run in parellel
 BASE_DIR=/home/laurence/work/code/FAB-TORCH/results
 TIMELIMIT=24
+N_FLOW_FORWARD_PASS = 50_000
```

Comment:
To test you can initially set this to a low number, do a run, and then increase it and see that it carries on from the previous checkpoint